packages/inline-definition/src/InlineDefinition.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tooltip, { TooltipProps } from '@leafygreen-ui/tooltip';
+import { Body } from '@leafygreen-ui/typography';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const underline = css`
+  display: inline-flex;
+  background-repeat: repeat-x;
+  background-position: center bottom;
+  background-size: 3px 2px;
+  transition: background-image 150ms ease-in-out;
```

Comment:
I'll be pleasantly shocked if I'm wrong here, but the background-image CSS property doesn't support transitions.

Reply:
nope you're right